bin/seeg-ct.sh

```diff
@@ -69,8 +69,8 @@ label_with_dilation("masked_CT.nii", "dilated_CT.nii", "labeled_CT.nii")
 EOF
 
 echo compare labels to implantation schema
-echo mrview T1.nii.gz -overlay.load=dilated_CT.nii
-echo $open $implantation_pptx
+echo mrview T1.nii.gz -overlay.load labeled_CT.nii -overlay.interpolation_off -overlay.colourmap 3
+echo open implantation_pptx
```

Comment:
This is what I was thinking was the intention, but it seemed unnecessary to me to force the user to set the variables just to print it back to him.